Tests/SwiftFormatPrettyPrintTests/SemicolonTests.swift

```diff
@@ -0,0 +1,24 @@
+public class SemiColonTypeTests: PrettyPrintTestCase {
```

Comment:
Can you add a couple test cases like the ones you already have, but with the members wrapped in a type? For example,

```swift
struct Foo {
  var foo = false; var bar = true; var baz = false
}
```

```swift
struct Foo {
  var foo = false
  var bar = true
  var baz = false
}
```

In the existing tests, since the `var`s aren't direct children of a type declaration, they'll be simple `CodeBlockItem`s rather than `MemberDeclListItem`s, so we don't have any test coverage for that part yet.

Reply:
I don't know why I didn't do this... Fixed the test cases!